app/src/main/AndroidManifest.xml

```diff
@@ -33,6 +35,10 @@
             android:theme="@style/AppTheme"
             android:windowSoftInputMode="stateAlwaysHidden" />
 
+        <activity
+            android:name=".ui.home.HomeActivity"
```

Comment:
Što sve misliš imati u ovom Activitiju?

Reply:
To je u biti kao main activity, fragmenti ce se tu izmijenjivati i navigation drawer je unutra

Reply:
A mainActivity je "preLogin"